.github/workflows/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+
+## Description
+
+<!--
+Please describe your change and provide as much context as needed including a link to the reference issue / ticket / Trello card. 
+-->
+
+* How to test the change: <description or reference to the Acceptance Criteria>
```

Comment:
The "reference to the Acceptance Criteria" doesn't make much sense outside of our team 😄 But other than that, I would move the "How to test" into its own section, because a lot of times a step by step guide is needed and it can take up some space.